packages/x-charts-pro/src/context/ZoomProvider/Zoom.types.ts

```diff
@@ -64,8 +118,16 @@ export type ZoomData = {
 };
 
 export type ZoomProps = {
+  /**
+   * The zoom data of type [[ZoomData]] which lists the zoom data related to each axis.
```

Comment:
Why using `[[ ... ]]`? This syntax works as follow: If an interface with the same name is documented, it will be replace by a link to this page.

To get it to work, you need to require the documentation of this interface by modifying these lines:

https://github.com/mui/mui-x/blob/3e0c2fd6754602f7e004d1313a3292efcb6c50ae/docs/scripts/createXTypeScriptProjects.ts#L156-L167 

It's useful for complex object. For example `AxisConfig`. But the `ZoomData` is for now an object with only 3 properties. Not sure it worths a dedicated page